### ISSUE TITLE
panel applets duplicated after testing extra gcard setup

### ISSUE THREAD

**Issue by sunweaver** [mate-desktop/mate-panel#353]
Hi,

we had just tested a setup where two monitors were connected to two different gcards (DVI ports) on the same system.

  DISPLAY=:0.0 (Intel, onboard Graphics)
  DISPLAY=:0.1 (NVIDIA, PICE)

The primary screen (:0.0) looked like a normal mate desktop. The secondary screen (:0.1) had two empty panels (one at the top, one at the bottom).

After we had found out, that it is not possible to drag a window from one screen to the other (because of the different hardware characteristics, I guess), we then switched back the setup to DVI and HDMI via the PCIE NVIDIA gcard.

Now, here comes the bug: After switching back to DVI+HDMI via NVIDIA PCIE card, all mate-panel applets had been duplicated. We then tried to remove superfluous applets, but removing a redundant applet from the panel did remove both applets. To get things straight the fast way, we dropped the complete DConf configuration.

Greets,
Mike

**Comment by monsta** [comment]
Does it look like on the screenshot in https://github.com/mate-desktop/mate-session-manager/issues/82?

**Comment by sunweaver** [comment]
Yep! It looked similar to what the referenced screen shot shows.

**Comment by monsta** [comment]
Ok, I closed that one since it was in the wrong repo, let's leave this one open. Copied the link to screenshot here for convenience.

![double vision](https://cloud.githubusercontent.com/assets/3869928/7357493/bcb9d1c2-ecf4-11e4-921e-673b27807824.png)

**Comment by raveit65** [comment]
Does this issue still exists?
